bundles/org.openhab.binding.bticinosmarther/README.md

```diff
@@ -0,0 +1,328 @@
+# BTicino Smarther Binding
+
+The Smarther binding implements a bridge to the Legrand/BTicino Smarther v2.0 API and allows you to control your BTicino Smarther chronothermostat units with openHAB, making it possible to discover Smarther units connected to your Legrand/BTicino Smarther account.
+
+Smarther chronothermostat is produced by BTicino (https://www.bticino.com/products-catalogue/smarther-the-connected-thermostat/), has its own API set and doesn't support the OpenWebNet protocol.
+
+Companion HABPanel widget can be found here: [BTicino Smarther HABPanel widget](https://github.com/MrRonfo/openhab-smarther-widget)
+
+# Configuring the binding
+
+The binding requires you to register an Application with Legrand Developer portal at https://developer.legrand.com
+This will get you a set of Client ID and Client Secret parameters to be used by your binding configuration.
```

Comment:
This is not really correct. The binding itself does not have ANY configuration in your case.
All you describe here is required for configuring a `BTicino Smarther Bridge`. Theoretically, you can have users with multiple applications registered at Legrand and using different bridges for them.

Reply:
Correct. See previous comment.